client/extensions/wp-super-cache/advanced-tab.jsx

```diff
@@ -16,37 +17,58 @@ import ExpiryTime from './expiry-time';
 import FixConfig from './fix-config';
 import LockDown from './lock-down';
 import Miscellaneous from './miscellaneous';
+import QueryNotices from './data/query-notices';
 import RejectedUserAgents from './rejected-user-agents';
 import WrapSettingsForm from './wrap-settings-form';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getNotices } from './state/notices/selectors';
 
 const AdvancedTab = ( {
 	fields: {
 		is_cache_enabled,
 		is_super_cache_enabled,
 	},
+	notices,
+	siteId,
 } ) => {
 	return (
 		<div>
+			<QueryNotices siteId={ siteId } />
 			<Caching />
-			<Miscellaneous />
+			<Miscellaneous notices={ notices } />
```

Comment:
It was coming from the HOC before - https://github.com/Automattic/wp-calypso/pull/13936/files#diff-07cc00eab7fb718e2f4a4dbe07abbb4dL312. Another case of the HOC doing too much. :smirk: